### ISSUE TITLE
New Update, more rejected shares!

### ISSUE THREAD

**Issue by Tanujshriyan** [trexminer/T-Rex#572]
Recently updated the miner to 0.21.6. Now I have more rejected shares than I ever had. 
(#552 I'm this guy!)

**Comment by trexminer** [comment]
When the miner starts, does it print out a red exclamation mark after generating DAG like so?
![image](https://user-images.githubusercontent.com/44735836/129433991-fb8a3b5f-d0ec-49c5-8493-a0201654925f.png)

**Comment by Tanujshriyan** [comment]
Couldn't check that but I haven't got any invalid shares after that. I think its fixed somehow by itself.

**Comment by mrjamiemoffat** [comment]
I just updated to 0.21.6 and I've had a lot of rejected shares since doing so.  I had started getting an increased amount of rejecteds on the 20.5 version - normally only 1 or 2 per day at most - around 2k shares.  On the old version I started hitting around 5% rejected. I've just updated today but I'm like 10/50 rejected so far.  Normally my ping is under 100 ms, but when the shares are failing it's like 10,000 + 

Any help would be appreciated
